src/environments/.env.ts

```
@@ -2,8 +2,8 @@
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '250703',
-    'hash': 'gc4224678'
+    'version': '250923',
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The 'version': '250923', key has incorrect delimiter

(IncorrectDelimiter)